tests/helpers/Participant.ts

```diff
@@ -383,4 +404,87 @@ export class Participant {
 
         return await localDisplayName.getText();
     }
+
+    /**
+     * Gets avatar SRC attribute for the one displayed on local video thumbnail.
+     */
+    async getLocalVideoAvatar() {
+        const avatar
+            = this.driver.$('//span[@id="localVideoContainer"]//img[contains(@class,"userAvatar")]');
+
+        return await avatar.isExisting() ? await avatar.getAttribute('src') : null;
+    }
+
+    /**
+     * Gets avatar SRC attribute for the one displayed on large video.
+     */
+    async getLargeVideoAvatar() {
+        const avatar = this.driver.$('//img[@id="dominantSpeakerAvatar"]');
+
+        return await avatar.isExisting() ? await avatar.getAttribute('src') : null;
+    }
+
+    /**
+     * Returns resource part of the JID of the user who is currently displayed in the large video area.
+     */
+    async getLargeVideoResource() {
+        return this.driver.execute(() => APP.UI.getLargeVideoID());
```

Comment:
no await?

Reply:
Execute is sync if I remember correctly

Reply:
I will double check

Reply:
If that's the case then drop the async from the function definition?

Reply:
Ok, will do. There is another function executeAsync when you want async.

Reply:
Yep, the example is with await https://webdriver.io/docs/api/browser/execute/